docs/source/tutorials/plots_intro.ipynb

```diff
@@ -530,6 +530,7 @@
    "id": "5211f6de-d447-46a8-8652-3c9fd77d7915",
```

Comment:
<p>This is a good point to make, but we could also pass <code>plot_kwargs = {'kde': {'zorder': 100}}</code> and somehow only apply that to <code>mu</code> via <code>pc_kwargs</code> I guess ? Might be a useful exercise to show, since after having read this, it's not immediately obvious to me how to do it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz-plots/pull/261/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>